src/graphorm/index.ts

```diff
@@ -245,6 +245,15 @@ const obj = new GraphORM({
             ELSE 0 END)
           `,
       },
+      referralToken: {
+        transform: (value: string, ctx: Context, member: SourceMember) => {
```

Comment:
Can we not simply use nullIfNotSameUser fn?

Reply:
Did not know this one exister, for sure 👌